Flick/Cells/ProfileSummaryTableViewCell.swift

```diff
@@ -55,17 +52,29 @@ class ProfileSummaryTableViewCell: UITableViewCell {
 
         setupConstraints()
     }
-    
-    private func setupConstraints() {
+
+    private func calculateFriendsPreviewWidth() -> Int {
         // Calculate width of friends preview based on number of friends and spacing between cells
-        let numFriendsInPreview = min(friends.count, 7) + 1
+        if friends.count == 0 { return 0 }
+        let numFriendsInPreview = min(friends.count, maxFriendsPreview) + 1
         let fullFriendsWidth = numFriendsInPreview * 20
         let overlapFriendsWidth = (numFriendsInPreview-1) * condensedCellSpacing * -1
         let friendsPreviewWidth = fullFriendsWidth - overlapFriendsWidth
+        return friendsPreviewWidth
+    }
 
-        let padding = 20
+    private func calculateUserInfoViewWidth(friendsPreviewWidth: Int) -> Int {
+        let padding = friends.count == 0 ? 0 : 20
         let userNameLabelWidth = Int(usernameLabel.frame.size.width)
         let userInfoViewWidth = userNameLabelWidth + padding + friendsPreviewWidth
+        return userInfoViewWidth
+    }
```

Comment:
i'm gonna make an issue for possibly moving calculation of width to `UserPreviewView` to reduce redundant code, since it's used in several places in the app